storage/migrations/22_runtime_abi.up.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+
+ALTER TABLE chain.runtime_transactions
+    -- For evm.Call transactions, we store both the name of the function and
+    -- the function paramters.
+    ADD COLUMN evm_fn_name TEXT,
+    ADD COLUMN evm_fn_params JSONB,
```

Comment:
Hm actually the JSONB will have a list of (k,v) tuples, where k is the param name and v is the param value. It shouldn't need to be JOINed with the abi since we do that during event parsing already.

From my understanding the mapping of event param names to values should be 1:1. Warren's [ParseEvent](https://github.com/oasisprotocol/nexus/blob/main/analyzer/runtime/abiparse/abiparse.go#L109) extracts the input values whether they're stored in the topics or in the event data, which should be exhaustive. We fetch param names from the [Event](https://pkg.go.dev/github.com/ethereum/go-ethereum/accounts/abi#Event) Inputs, which should also be exhaustive since go eth will [autogenerate names for unnamed args](https://github.com/ethereum/go-ethereum/blob/v1.13.5/accounts/abi/event.go#L70).